### PR TITLE
add {{.RunningFor}} placeholder in ps --format

### PR DESCRIPTION
**Pull request by ashley-cui**
For docker compatibility

#5535

Signed-off-by: Ashley Cui <acui@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Could you add a test?

**Comment by ashley-cui** [comment]
@rhatdan what would i check the output to be, since time moves? or should i just check that it doesnt fail

**Comment by rhatdan** [comment]
/approve
LGTM
@mheon @TomSweeneyRedHat @giuseppe @vrothberg @QiWang19 @jwhonce PTAL

**Comment by TomSweeneyRedHat** [comment]
LGTM
assuming happy tests

**Comment by mheon** [comment]
Could you look for "Running for" in the output, maybe? Or "seconds"?

**Comment by mheon** [comment]
Ah, the existing test looks fine.

LGTM

File: test/e2e/ps_test.go [hunk 1]
```diff
@@ -476,5 +476,13 @@ var _ = Describe("Podman ps", func() {
...
 		Expect(session.OutputToString()).To(ContainSubstring("echo very long cr..."))
 	})
+	It("podman ps --format {{RunningFor}}", func() {
+		_, ec, _ := podmanTest.RunLsContainer("")
+		Expect(ec).To(Equal(0))
 
+		result := podmanTest.Podman([]string{"ps", "-a", "--format", "{{.RunningFor}}"})
+		result.WaitWithDefaultTimeout()
+		Expect(result.ExitCode()).To(Equal(0))
+		Expect(result.OutputToString()).To(ContainSubstring("ago"))
```
Comment (edsantiago) [inline test/e2e/ps_test.go:486]: Is there an `EndsWith()` function in ginkgo? If so, for a future PR (absolutely not for this one, it's not worth the CI respin cost), might I suggest `EndsWith(" ago")`? Note the space, too. The current test would match on "paragon" and "agony" - not that those are likely to appear in this context, but hey.

**Comment by mheon** [comment]
/hold

One test flaked.

**Pull request merged**